apps/cyberstorm-nextjs/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN chown node:node -R /app
 
 USER node
 RUN --mount=type=secret,id=npmrc,target=/home/node/.npmrc,uid=1000 yarn install --frozen-lockfile
- RUN yarn workspace @thunderstore/cyberstorm-nextjs run build
+ RUN yarn workspace @thunderstore/cyberstorm-nextjs run build:dev
```

Comment:
Even if this is "just get something deployed" attempt, I would think we don't want to deploy to production using the local development env variable file, but rather the actual environment variables. This is the reason I mentioned resorting to hacky pre-building in Discord.